site/app/views/NavigationView.php

```diff
@@ -252,7 +253,33 @@ public function showGradeables($sections_to_list) {
                     $gradeable_title = '<label>'.$g_data->getName().'</label><a class="external" href="'.$g_data->getInstructionsURL().'" target="_blank"><i style="margin-left: 10px;" class="fa fa-external-link"></i></a>';
                 }
                 else{
-                    $gradeable_title = '<label>'.$g_data->getName().'</label>';
+                    if ($g_data->getType() == GradeableType::ELECTRONIC_FILE) {
```

Comment:
What about the other types of gradeable?  They should be deleteable also, if they have no ta grade data entered for any student.